packages/storage/index.js

```diff
@@ -1,4 +1,8 @@
-export const put = (key, value) => window.localStorage.setItem(key, value);
-export const get = key => window.localStorage.getItem(key);
-export const remove = key => window.localStorage.removeItem(key);
-export const clear = () => window.localStorage.clear();
+export const put = (key, value, { storage = window.localStorage } = {}) =>
```

Comment:
Dont we have a storage package somewhere? Maybe then we actually do not need that and should:

1, Extend our package we have
2. Just operate with those APIs straight away